string status
int16 action
